Makefile

```diff
@@ -46,12 +49,12 @@ cluster/clean:
 	-kubectl delete -f deploy/role.yaml
 	-kubectl delete -n $(NAMESPACE) -f deploy/role_binding.yaml
 	-kubectl delete -n $(NAMESPACE) -f deploy/service_account.yaml
-	-kubectl delete -n $(NAMESPACE) -f deploy/crds/push_v1alpha1_unifiedpushserver_crd.yaml
```

Comment:
This should be put back I think

Reply:
it is not deleted just move, please could you check the file?

Reply:
Done.